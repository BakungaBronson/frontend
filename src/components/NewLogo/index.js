import React from 'react';
import { Link } from 'react-router-dom';
import { ReactComponent as NewLogoIcon } from '../../assets/images/NewLogo.svg';
import './NewLogo.css';

const NewLogo = () => {
  return (
    <Link to="/" className="LogoWrapper">
      <div className="Logo">
        <NewLogoIcon className="NewLogoIcon" />
        <div className="LogoType">Crane Cloud</div>
      </div>
    </Link>
  );
};

export default NewLogo;
