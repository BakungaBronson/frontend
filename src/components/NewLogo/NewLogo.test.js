import React from 'react';

import { shallow } from 'enzyme';

import NewLogo from './index';

describe('Test new logo', () => {
    it('checks whether the component rendered correctly', () => {
        const LogoComponent = shallow(<NewLogo />);
        
        expect(LogoComponent).toMatchSnapshot();

        const BrandLogoType = LogoComponent.find("div > div");
        expect(BrandLogoType.hasClass("BrandLogoType")).toEqual(true);
        expect(BrandLogoType.text()).toBe("Crane Cloud");
    })
})