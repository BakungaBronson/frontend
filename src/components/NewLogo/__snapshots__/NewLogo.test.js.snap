// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test new logo checks whether the component rendered correctly 1`] = `ShallowWrapper {}`;
